moduleframework/dockerlinter.py

```diff
@@ -173,18 +173,24 @@ def check_baseruntime(self):
         if FROM in self.docker_dict:
             return [x for x in self.docker_dict[FROM] if "baseruntime/baseruntime" in x]
 
-    def check_microdnf(self):
+    def check_dnf_nodocs_flag(self):
         """
         Function returns docker labels
         :return: label dictionary
         """
         if RUN in self.docker_dict:
-            for val in self.docker_dict[RUN]:
-                if val.startswith("yum") or " yum " in val:
-                    return False
-                if val.startswith("dnf") or " dnf " in val:
-                    return False
-                else:
-                    return True
-
+            if '--nodocs' in self.docker_dict[RUN]:
```

Comment:
I kind of don't understand the meaning here: what if the dockerfile is meant for centos and there is no microdnf in centos, hence the test would fail. What is the point of creating a test which will fail always on a selected platform (=use case).